docs/vz235_mellow/top_cover/top_cover.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+title: 8.1 Top cover
+parent: 8. Top cover and doors panels
+grand_parent: Vz235 - Mellow Kit
+has_toc: false
+nav_order: 1
+has_children: false
+permalink: /vz235_mellow/top_cover/latches
```

Comment:
The permalink here doesn't match the name of the file, which would be recommended. 

You can replace `permalink:` with `slug:` to define the name of one page, with the rest of the navigation structure determined by the folder names.

Another option is leaving both out entirely.

Both `has_toc` as well as `has_children` can be left out if they are false; we do not have them true by default and they are false by default according to the JustTheDocs defaults. These pages can't have children as of now, regardless. Two levels is the max.

Reply:
updated